src/components/teamBoxes/TeamOneBox.jsx

```diff
@@ -161,4 +177,4 @@ const RedTeamBox = () => {
   );
 };
 
-export default RedTeamBox;
+export default TeamOneBox;
```

Comment:
This doesn't have to go in this PR but - just realized - we will probably want to change the .css file names so they are called team1 and not red